src/sinol_make/commands/doc/__init__.py

```diff
@@ -56,11 +70,28 @@ def configure_subparser(self, subparser: argparse.ArgumentParser):
             help='Compile latex files to pdf',
             description='Compiles latex files to pdf. By default compiles all files in the `doc` directory.\n'
                         'You can also specify files to compile.')
+        parser.add_argument('--latex-compiler', dest='latex_compiler', choices=['auto', 'pdflatex', 'latex_dvi'],
+                            help='Compiler used to compile documents (default: auto)', default='auto')
```

Comment:
Maybe you could add an explanation which compiler should be used and what `auto` does?

Reply:
Directly in help field, or in other way?

Reply:
In the help field